runtime/fundamentals/web_standards.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Web Standards and Node APIs"
+---
+
+Deno aims to bridge the gap between the browser and the backend with a set of
+web standard APIs that are familiar to web developers. These APIs are designed
```

Comment:
this sounds a little roundabout- maybe something along these lines: "Deno aims to narrow the gap between browser and server-side JavaScript by providing the exact same web standard APIs available client side in the server. This means if you know how to program a browser, you already know how to program Deno."